desc/compute/_equil.py

```diff
@@ -843,3 +843,39 @@ def _P_ISS04(params, transforms, profiles, data, **kwargs):
         )
     ) ** (1 / 0.39)
     return data
+
+
+@register_compute_fun(
+    name="P_fusion",
+    label="P_{fusion}",
+    units="W",
+    units_long="Watts",
+    description="Fusion power",
+    dim=0,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="",
+    data=["rho", "ni", "<sigma*nu>", "sqrt(g)"],
+    resolution_requirement="rtz",
+    reaction="str: Fusion reaction. One of {'T(d,n)4He', 'D(d,p)T', 'D(d,n)3He'}. "
+    + "Default is 'T(d,n)4He'.",
+)
+def _P_fusion(params, transforms, profiles, data, **kwargs):
+    reaction = kwargs.get("fuel", "T(d,n)4He")
+    match reaction:
+        case "T(d,n)4He":
+            energy = 3.52e6 + 14.06e6  # eV
+        case "D(d,p)T":
+            energy = 1.01e6 + 3.02e6  # eV
+        case "D(d,n)3He":
+            energy = 0.82e6 + 2.45e6  # eV
```

Comment:
OK I switched this to `fuel` instead of `reaction`. For now let's only implement DT, and we can worry about adding more complicated fuels later if people want them. 

One idea I though of is adding an additional kwarg to allow the user to specify the energy themselves if they want to include extra reactions between products (and if not supplied it would default to the basic case for that fuel). But I don't love adding lots of extra compute kwargs